src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -248,11 +248,16 @@ class Breakpoints extends PureComponent {
       click: () => setBreakpointCondition(breakpoint.location)
     };
 
-    const editCondition = {
+    const addOrEditCondition = {
       id: "node-menu-edit-condition",
-      label: editConditionLabel,
-      accesskey: editConditionKey,
-      click: () => toggleConditionalBreakpointPanel(breakpoint.location.line)
+      label: addOrEditConditionLabel,
+      accesskey: addOrEditConditionKey,
+      click: () => {
+        const sourceId = breakpoint.location.sourceId;
+        const line = breakpoint.location.line;
+        this.props.selectSource(sourceId, { line });
```

Comment:
yes! good catch. Perhaps we move this invokation to this action: `toggleConditionalBreakpointPanel`